src/Stryker.Core/Stryker.Core/TestRunners/TestRunnerFactory.cs

```diff
@@ -25,13 +26,13 @@ public ITestRunner Create(StrykerOptions options, ProjectInfo projectInfo)
             switch (options.TestRunner)
             {
                 case TestRunner.DotnetTest:
-                    testRunner = new DotnetTestRunner(options.BasePath, new ProcessExecutor(), new TotalNumberOfTestsParser());
+                    testRunner = new DotnetTestRunner(Path.GetDirectoryName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath), new ProcessExecutor(), new TotalNumberOfTestsParser());
```

Comment:
I would prefer the callee becomes responsible for correctly handling getting the directory name from the path since we are dealing with strings and have no mechanism to validate the string input so cannot force the caller to make the right decision.